src/pmi/errnames.txt

```diff
@@ -42,6 +42,8 @@
 **pmi_invalid_clique_size %d:PMI_Get_clique_ranks returned an invalid size %d
 **pmi_job_getid:PMI2_Job_GetId failed
 **pmi_job_getid %d:PMI2_Job_GetId returned %d
+**pmi_kvs_get_parent_port:PMI_KVS_Get parent port failed
+**pmi_kvs_get_parent_port %d:PMI_KVS_Get parent port failed %d
```

Comment:
I think what we needed is a more general 2-param message: `**pmi_kvs_get %s %d: PMI_KVS_Get %s failed: %d`

Reply:
We don't need it. The `%d` is for the error code, not the parameter.

Reply:
Are we going to create separate error message for all key names? If you create the 2-param message, then it can be used for all `PMI_KVS_Get` errors.

Reply:
I think it is better to clean it up when refactoring the PMI related code.